.gitignore

```diff
@@ -1,8 +1,8 @@
 coverage/
 data
-lib/
+/lib/
 node_modules/
 public/scripts/
 public/styles/
 public/templates/
-spec/
+/spec/
```

Comment:
Fantastic find, have adjusted `blizzardry` and some other repos accordingly! :+1:

Could you prefix the other directories, too?